profiles/models.py

```diff
@@ -3,8 +3,12 @@
 # Create your models here.
 
 class User(AbstractUser):
```

Comment:
this is lousy usage of AbstractUser. 

* Learn how to inheritance work in python
* describe m2m relation with Word table by using UserWord